x/emissions/types/params.go

```diff
@@ -34,7 +36,7 @@ func DefaultParamsVersion() string {
 	return DefaultParams().Version
 }
 
-func DefaultParamsEpochLength() int64 {
+func DefaultParamsEpochLength() BLOCK_NUMBER {
```

Comment:
technically speaking this isn't a block number (in the sense of "block 62") but rather a delta of blocks (like "5 blocks") ...